### ISSUE TITLE
SVG files in projects don't load

### ISSUE THREAD

**Issue by nbriz** [netizenorg/netnet.studio#118]
**tl;dr** For some reason when working on a project with img elements pointing to svg files (uploaded to the project as an asset) they don't seem to load. 
<hr>

when working on a project (ie. a GitHub repo) netnet will inject a new "root" URL (ie. prepend the following to any part of the code using a local path: `api/github/proxy?url=https://raw.githubusercontent.com/[USER]/[REPO]/[BRANCH]`), this is handled in part by the netitor [here](https://github.com/netizenorg/netitor/blob/b34adb0c0434a1a1e57a8d8f6cab8bde0140a90d/src/main.js#L513) and in part by netnet (to handle the http proxy part) for example  [here](https://github.com/netizenorg/netnet.studio/blob/94ae1d5941d437f743c130e68b77f7728b14c67e/www/js/utils.js#L190) and [here](https://github.com/netizenorg/netnet.studio/blob/94ae1d5941d437f743c130e68b77f7728b14c67e/www/widgets/StudentSession.js#L198).

typically this works perfectly fine, for example when working on a project with a file called 'lil-bub.jpg' uploaded as an asset like this:

```html
<img src="lil-bub.jpg" alt="description of image">
```
netnet will inject the proper URL && what actually gets rendered in the output is:

```html
<img src="api/github/proxy?url=https://raw.githubusercontent.com/nbriz/font-demo/main/lil-bub.jpg" alt="description of image">
```
because this is a png (not an svg) this works fine. but if we were to do this:
```html
<img src="mouth-dot.svg" alt="description of image">
```

...which would then become:

```html
<img src="api/github/proxy?url=https://raw.githubusercontent.com/nbriz/font-demo/main/mouth-dot.svg" alt="description of image">
```
this does not work fine :(

when i take a look at the browsers dev tools, it seems the file is downloaded (the Network request come back 200) but when i take a look at the Inspector it says "could not load the image")

but strangely, if we write the following absolute path into our netnet code, it *does* work... wtf?

```html
<img src="https://raw.githubusercontent.com/nbriz/font-demo/main/mouth-dot.svg" alt="description of image">
```
